src/main/java/draylar/identity/screen/IdentityGui.java

```diff
@@ -0,0 +1,146 @@
+package draylar.identity.screen;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.text.LiteralText;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.util.Window;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.fabric.api.util.TriState;
+
+import io.github.cottonmc.cotton.gui.client.LightweightGuiDescription;
+import io.github.cottonmc.cotton.gui.widget.WButton;
+import io.github.cottonmc.cotton.gui.widget.WPlainPanel;
+import io.github.cottonmc.cotton.gui.widget.WScrollPanel;
+import io.github.cottonmc.cotton.gui.widget.WSprite;
+import io.github.cottonmc.cotton.gui.widget.WLabel;
+import io.github.cottonmc.cotton.gui.client.BackgroundPainter;
+
+import draylar.identity.cca.IdentityComponent;
+import draylar.identity.cca.UnlockedIdentitysComponent;
+import draylar.identity.screen.widget.WEntityButton;
+import draylar.identity.screen.widget.WPlayerButton;
+import draylar.identity.screen.widget.WSearchBar;
+import draylar.identity.registry.Components;
+
+public class IdentityGui extends LightweightGuiDescription {
+    private final List<LivingEntity> renderEntities = new ArrayList<>();
+    private List<WEntityButton> renderedButtons = new ArrayList<>();
+    public IdentityGui() {
```

Comment:
Formatting changes, newline before fields in classes, and newlines splitting methods

```suggestion
public class IdentityGui extends LightweightGuiDescription {

    private final List<LivingEntity> renderEntities = new ArrayList<>();
    private List<WEntityButton> renderedButtons = new ArrayList<>();
    
    public IdentityGui() {
```